api-client/.gitignore

```diff
@@ -108,6 +108,9 @@ cmake-build-release/
 # IntelliJ
 out/
 
+# VSCode
+.vscode/
```

Comment:
Probably it is enough to exclude `.vscode/settings.json`. What do you think @Znippet ?